### PR TITLE
RYA-325 Renamed rya.console to rya.shell.

### PR DESCRIPTION
**Pull request by jdasch**
## Description
Renamed rya.console to rya.shell.

### Tests
N/A

### Links
[Jira](https://issues.apache.org/jira/browse/RYA-325)

### Checklist
- [ ] Code Review
- [ ] Squash Commits

#### People To Reivew
@isper3at 
@meiercaleb

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/348/

File: extras/pom.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@ under the License.
         <module>rya.prospector</module>
         <module>rya.manual</module>
         <module>rya.periodic.service</module>
-        <module>rya.console</module>
+        <module>shell</module>
```
Comment (amihalik) [inline extras/pom.xml]: agree with Andrew.  Why not put this is rya.shell?

Comment (jdasch) [inline extras/pom.xml]: It makes filesystem navigation by shell faster and yields shorter, less redundant paths.  If that convention was fully expanded, we would end up having:
`incubator-rya/rya.extras/rya.shell` which is more rya's than I'd like to type.

Does windows still have those long path limitations?

Comment (amihalik) [inline extras/pom.xml]: okay, I'm fine with this.

**Pull request closed without merging**